<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>环境监测</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(193, 125, 17);
font: 75 22pt &quot;Ubuntu Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>当前环境信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);</string>
     </property>
     <property name="text">
      <string>LUX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);</string>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>290</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/back7.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>190</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/back7.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>210</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/back7.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);</string>
     </property>
     <property name="text">
      <string>温度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);

</string>
     </property>
     <property name="text">
      <string>  光照：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>60</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Ubuntu&quot;;
color: rgb(193, 125, 17);</string>
     </property>
     <property name="text">
      <string>湿度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/back4.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_data">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label_10</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_7</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>lcdNumber_2</zorder>
    <zorder>lcdNumber_3</zorder>
    <zorder>lcdNumber</zorder>
    <zorder>label_data</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>设备控制</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(192, 218, 234);
       color: black;                         
       font-weight: bold;      </string>
     </property>
     <property name="text">
      <string>关闭界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BeeButton_on">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {     
background-color: rgb(192, 218, 234);   
       color: gray;                 
       font-weight: normal;
    }
  QPushButton:disabled { 
background-color: rgb(192, 218, 234);
       color: black;                         
       font-weight: bold;      
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="FanButton_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {     
background-color: rgb(192, 218, 234);   
       color: gray;                 
       font-weight: normal;
    }
  QPushButton:disabled { 
background-color: rgb(192, 218, 234);
       color: black;                         
       font-weight: bold;      
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="MonButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(192, 218, 234);
       color: black;                         
       font-weight: bold;      </string>
     </property>
     <property name="text">
      <string>打开监控</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>861</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/bg2.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>电灯1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;
</string>
     </property>
     <property name="text">
      <string>电灯2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>150</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;
</string>
     </property>
     <property name="text">
      <string>电灯3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>208</width>
       <height>136</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/jk.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>风扇</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>警报</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>50</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>CloseButton</zorder>
    <zorder>BeeButton_on</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>FanButton_1</zorder>
    <zorder>pushButton</zorder>
    <zorder>MonButton</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
    <zorder>label_18</zorder>
    <zorder>label_21</zorder>
    <zorder>label_20</zorder>
    <zorder>label_22</zorder>
    <zorder>label_23</zorder>
    <zorder>label_35</zorder>
    <zorder>label_36</zorder>
    <zorder>label_37</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>修改监测阈值</string>
    </attribute>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>80</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>一级照明</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LED1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>二级照明</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Hum">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>220</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(192, 218, 234);
border-radius:14px;
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LED灯</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_Ctrol">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>451</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal { 
                height: 8px; 
                background: #4285F4; 
                border-radius: 4px; 
                margin: 0 10px; 
            }
            QSlider::handle:horizontal { 
                background: white; 
                border: 1px solid #4285F4; 
                width: 16px; 
                margin: -4px 0; 
                border-radius: 8px; 
            }</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(192, 218, 234);
border-radius:14px;
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>风扇开启温度</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(192, 218, 234);
border-radius:14px;
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/buzzer.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_SmartOn">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>300</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>开启</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>0</y>
       <width>861</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/bg3.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>蜂鸣器开启湿度</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_SmartOff">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>三级照明</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(192, 218, 234);
font: 14pt &quot;Ubuntu&quot;;
border-radius:14px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>310</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/pic.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/fan.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(192, 218, 234);
border-radius:14px;
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>label_29</zorder>
    <zorder>label_26</zorder>
    <zorder>label_30</zorder>
    <zorder>textEdit_Hum</zorder>
    <zorder>label_28</zorder>
    <zorder>slider_Ctrol</zorder>
    <zorder>textEdit_4</zorder>
    <zorder>label_32</zorder>
    <zorder>textEdit_2</zorder>
    <zorder>label_33</zorder>
    <zorder>radioButton_SmartOn</zorder>
    <zorder>label_34</zorder>
    <zorder>radioButton_SmartOff</zorder>
    <zorder>label_31</zorder>
    <zorder>textEdit</zorder>
    <zorder>label_25</zorder>
    <zorder>label_27</zorder>
    <zorder>textEdit_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
